.vscode/defsettings.json

```diff
@@ -1,7 +0,0 @@
-{
```

Comment:
Why was this file deleted?

Reply:
don't recall removing it. will add it back